File: src/HW1/Task2.java
//Императивное программирование:
//Задача 2: Поиск наименьшего числа в списке.
// Напишите программу, которая находит наименьшее число в заданном списке с помощью цикла.

package HW1;

public class Task2 {
    public static void main(String[] args) {
        int[] array = new int[] {2, 5, 9, 1, 6};
        System.out.println("Наименьшее число в списке: " + MinNum(array));
    }

    public static int MinNum (int[] arr){
        int minNum = arr[0];
        for (int i = 0; i < arr.length; i++) {
            if (arr[i] < minNum) {
                minNum = arr[i];
            }
        }
        return minNum;
    }
}

File: src/HW1/task_2.py
# Императивное программирование:
# /Задача 2: Поиск наименьшего числа в списке.
# Напишите программу, которая находит наименьшее число в заданном списке с помощью цикла.

array = [2, 5, 9, 1, 6]

min_num = array[0]

for i in range(len(array)):
    if array[i] < min_num:
        min_num = array[i]

print(f"Наименьшее число в списке: {min_num}")
